src/core/types/function/index.js

```diff
@@ -14,14 +16,17 @@
  */
 export function once(fn) {
   let evaluated = false;
+  /** @type {T?} */
   let retValue = null;
   let callback = fn;
+
   return (...args) => {
     if (!evaluated) {
       retValue = callback.apply(self, args);
       evaluated = true;
       /** @type {?} */ (callback) = null; // GC
     }
+    devAssert(retValue);
```

Comment:
Is this correct? Can't `callback` return something falsey, since `T` could be equivalent to `?string` or `boolean` or something similar?